scripts/expect/telnet.exp

```diff
@@ -4,10 +4,44 @@ source [file join [file dirname [info script]] framework/test_core.exp]
 
 namespace import autotest::*
 
-TEST_CASE {execute "help" command through telnet} {
+TEST_SUITE_SETUP
+
+TEST_CASE_TARGET {Execute "help" command once} {
+	set TELNET_PROMPT ":/#"
+
+	send "help\r"
+	test_assert_regexp_equal "Available commands" $TELNET_PROMPT
+
+	return 0
+}
+
+TEST_CASE_TARGET {Execute "help" multiple times} {
 	set TELNET_PROMPT ":/#"
 
-	test_assert_regexp_equal "help\r\n"    $TELNET_PROMPT
+	for {set i 0} {$i < 128} {incr i} {
+		send "help\r"
+		test_assert_regexp_equal "Available commands" $TELNET_PROMPT
+		exec sleep 0.1
+	}
 
 	return 0
 }
+
+if {0} {
```

Comment:
if 0?

Reply:
It was intentionally disabled in https://github.com/embox/embox/pull/1260/commits/69ca1bb6a8c159393f4d75c05be17a42c53df60e since this test failed (but test is correct by itself).